tsconfig.base.json

```diff
@@ -23,7 +23,7 @@
     "strictFunctionTypes": true,
     "strictNullChecks": true,
     "strictPropertyInitialization": true,
-    "target": "ES2019",
+    "target": "ES2022",
```

Comment:
Unlike Puppeteer we can bump the version here as we only use the normal Private properties.